announcements/LL_230_changelog.md

```diff
@@ -0,0 +1,222 @@
+- renamed the `infoURLs` and `vidURLs` fields in the `Launch` model to `info_urls` and `vid_urls`
+- renamed the `location` field in the `Landing` model to `landing_location`
+- renamed the `manufacturer` filter on the `SpacecraftConfiguration` endpoint to `agency`, to match the filtering field
+- renamed the `nation_url` field in the `Agency` model to `social_icon`
+
+### ⚙️ Type & format changes
+
+- made the `founding_year` field an int in the `Agency` model
+- made the `latitude` and `longitude` fields floats in the `Location` and `Pad` models
+- made the `spacecraft_stage` field in the `Rocket` model an M2M relation to the `SpacecraftFlight` model
+- made the `status`, `net`, `window_start` and `window_end` fields non-nullable in the `Launch` model
+- made the `status` field in the `Launcher` model a `LauncherStatus` FK, and added filtering by FK ID on the endpoint
+- made the `type` field in the `Agency` model a FK to the `AgencyType` config
+- replaced the `agency_id` field in the `Pad` model with an M2M relation to the `Agency` model
+- replaced the `turn_around_time_days` int field in the `FirstStage` model by an ISO8601-formatted `turn_around_time`
+  duration field
+- reworked the `DockingEvent` model to make it work with a chaser and a target, both of which can be
+  a `Spacecraft`, `SpaceStation`, or `Payload` FK
+- reworked the `DockingLocation` model to make it have both a `Spacecraft` FK and a `Payload` FK
+
+### 🗑️ Field removals
+
+- removed the `holdreason` field from the `Launch` model
+- removed the `launch_library_url` fields from all models where it was still present (remains from the LL1 era)
+- removed the `news_url` and `video_url` fields from the `Event` model (replaced by `InfoURL` and `VidURL`
+  M2Ms like for launches, as introduced in 2.2.0)
+- removed the `r_spacex_api_id` field from the `Launch` model and the associated filter on the endpoint (API deprecated)
+- removed the `the nation_url` field from the `SpacecraftConfiguration` model
+- removed the `twitter` and `instagram` fields from the `Astronaut` model
+- removed the `vehicle_range` field from the `LauncherConfiguration` model
+
+### 🚀 Misc
+
+- added the `Payload Deployed` (ID 9) launch status, returned as `In Flight` (ID 6) in older versions
+
+### 📈 API & endpoint enhancements
+
+- automated the endpoint docs to provide fully exhaustive filtering, sorting, ordering, and generic information
+- major refactor of all serializers to clean up code and allow full tests coverage, this means some fields might have
+  been added/removed to/from the `list` and `normal` modes
+- major refactor all endpoints to ensure polymorphism is properly documented in the schema
+- removed the non-strict LSP & location filtering option from the `Launch` endpoint that prevented proper filter
+  documentation
+- renamed all endpoint URLs to use plural and `snake_case`
+- significant database-related performance improvements on all endpoints
```

Comment:
we could probably also name the validations (a.k.a drunk librarian protections) as an improvement here, so the API has less incorrect data

Reply:
done